doc/1/controllers/server/get-stats/index.md

```diff
@@ -0,0 +1,39 @@
+---
+code: true
+type: page
+title: GetStatsAsync
+description: Returns statistics snapshots within a provided timestamp range.
+---
+
+# GetStatsAsync
+
+Returns statistics snapshots within a provided timestamp range.
+By default, snapshots are made every 10 seconds and they are stored for 1 hour.
+
+These statistics include:
+
+- the number of connected users per protocol (not available for all protocols)
+- the number of ongoing requests
+- the number of completed requests since the last frame
+- the number of failed requests since the last frame
+
+## Signature
+
+```csharp
+async Task<JObject> GetStatsAsync(Int64 start, Int64 end)
+```
+
+## Arguments
+
+| Arguments | Type | Description                      | Required |
```

Comment:
I don't think the field `Required` is relevant since you're forced to pass a  `start` and an `end` in the `GetStatsAsync` method and there is no other parameters.